auth/basic_auth.py

```
@@ -0,0 +1,92 @@
+
+class BasicAuth:
+    """Basic authentication with user management"""
+
+    def __init__(self):
+        self.users_file = "auth/users.json"
+        self.users = self._load_users()
+
+    def _load_users(self) -> Dict:
+        """Load users from file"""
+        if os.path.exists(self.users_file):
+            with open(self.users_file, "r") as f:
```

Ruff (0.11.9)

Unnecessary mode argument

Remove mode argument

(UP015)